apps/token/src/routes/home/index.tsx

```diff
@@ -1,147 +1,199 @@
+                <Link to={Routes.VALIDATORS}>
+                  <RoundedWrapper paddingBottom={true}>
+                    <div className="flex flex-col items-center m-[-1rem] px-4 py-6 hover:bg-neutral-800">
+                      <span className="text-5xl">{length}</span>
+                      <span className="text-sm uppercase text-neutral-400">
+                        {title}
+                      </span>
+                    </div>
+                  </RoundedWrapper>
+                </Link>
+              </div>
+            ))}
+
+            {trimmedActiveNodes.map(({ id, avatarUrl, name }) => (
+              <div key={id} className="col-span-2">
+                <Link to={`${Routes.VALIDATORS}/${id}`}>
+                  <RoundedWrapper paddingBottom={true} border={false}>
+                    <div className="flex items-center justify-center m-[-1rem] p-4 bg-neutral-900 hover:bg-neutral-800">
+                      {avatarUrl && (
+                        <img
+                          className="h-6 w-6 rounded-full mr-2"
+                          src={avatarUrl}
+                          alt={`Avatar icon for ${name}`}
+                        />
+                      )}
+                      <span className="text-sm">{name}</span>
+                    </div>
+                  </RoundedWrapper>
+                </Link>
+              </div>
+            ))}
+          </div>
+
+          {activeNodes.length > nodesToShow && (
+            <Link to={Routes.STAKING}>
+              <span className="underline">
+                And {activeNodes.length - nodesToShow} more...
+              </span>
+            </Link>
           )}
-        </p>
-        <p>
-          <a
-            data-testid="get-vega-wallet-link"
-            href={ExternalLinks.VEGA_WALLET_URL}
-            className="underline text-white"
-            target="_blank"
-            rel="nofollow noreferrer"
-          >
-            {t('Get a Vega wallet')}
-          </a>
-        </p>
-        <p>
-          <Link
-            data-testid="associate-vega-tokens-link-on-homepage"
-            to={`${Routes.VALIDATORS}/associate`}
-            className="underline text-white"
-          >
-            {t('Associate VEGA tokens')}
-          </Link>
-        </p>
-      </HomeSection>
-      <div className="flex gap-12">
-        <div className="flex-1">
-          <HomeSection>
-            <SubHeading title={t('Staking')} />
-            <p>
-              {t(
-                'VEGA token holders can nominate a validator node and receive staking rewards.'
-              )}
-            </p>
-            <p>
-              <Link to={Routes.VALIDATORS}>
-                <Button size="md" data-testid="staking-button-on-homepage">
-                  {t('Nominate a validator')}
-                </Button>
-              </Link>
-            </p>
-          </HomeSection>
+        </section>
+      )}
+      <section className="grid grid-cols-2 gap-12 mb-16">
```

Comment:
chore: this is quite the wall of JSX, could we break the sections into their own components. Even if they are just in the same file for readability?